app/pb_kits/playbook/pb_label_value/docs/_label_value_details.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= pb_rails("label_value", props: {
+  variant: "details",
+  label: "Installer",
+  icon: "truck",
+  title: "JD Installations LLC",
+}) %>
+
+<br>
+
+<%= pb_rails("label_value", props: {
+  variant: "details",
+  label: "Project",
+  icon: "home",
+  description: "33-12345",
+  title: "Jefferson-Smith",
+}) %>
+
+<br>
+
+<%= pb_rails("label_value", props: {
+  variant: "details",
+  label: "Project",
+  icon: "home",
+  description: "33-12345",
+  title: "Jefferson-Smith",
+  date: Date.new(2019, 11, 18)
+}) %>
+
+<br>
+
+<%= pb_rails("label_value", props: {
+  variant: "details",
+  active: true,
```

Comment:
Can we make a short write up for all our kits on how to use? I'm not sure on this one, and I think writing a quick doc will help define the logic (when to use what)

Reply:
I updated the description of the kit.

<img width="1378" alt="Screen Shot 2020-07-23 at 1 27 24 PM" src="https://user-images.githubusercontent.com/42459486/88318258-71af1500-cce8-11ea-8d4e-8c9cc23a36f8.png">